CHANGELOG.md

```diff
@@ -8,8 +8,6 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 > Place unreleased changes here.
 
-## [2.0.0.rc1](https://github.com/stellar/stellar-disbursement-platform-frontend/releases/tag/2.0.0-rc1)
```

Comment:
If we want to put the MTN changes under `unreleased` changes, should we also do that with backend repository ? https://github.com/stellar/stellar-disbursement-platform-backend/pull/260/files

Reply:
Sorry, that happened when I was solving some conflicts. Re-added the title in https://github.com/stellar/stellar-disbursement-platform-frontend/pull/93/commits/b7c5ef7e1428736a57320f2320326965f6cf1bb8.

I believe we should keep it there since the RC has been (pre) released.